CompilerSource/ipc/Path.fbs

```diff
@@ -0,0 +1,23 @@
+
+struct PathPoint {
+  x:int;
+  y:int;
+  speed:int;
+}
+
+table Path {
+  name:string;
+  id:int;
+
+  smooth:bool;
+  closed:bool;
+  precision:int;
+
+  //background_room_id:int; //This was used by LGM rather than ENIGMA; it doesn't need to be here.
+  snap_x:int; //TODO: Find out if these are needed and decide on keeping them
+  snap_y:int;

```

Comment:
Shall we dump these now rather than later in light of our discoveries about manual ids?

Reply:
Definitely extract these into their own message. Name it something like "PathEditorMetadata." I think they're still useful for EGM/GMX serialization. Actually, does GMX store these? That should be signal, right there.